components/automate-cli/cmd/chef-automate/infrastructure.go

```diff
@@ -151,16 +152,28 @@ func checkLicenseExpiry(licenseResult *LicenseResult) error {
 	}
 	licenseValidDate := time.Unix(licenseResult.Result.ExpirationDate.Seconds, 0) //gives unix time stamp in utc
 
-	//If the license type is commercial, adding grace period of 1 week
+	//If the license type is commercial, adding grace period of 1 month
+	licenseResult.Result.GracePeriod = false
 	if licenseResult.Result.LicenseType == commercial {
-		//Adding grace period for 7 days i.e. one week
-		licenseValidDate = licenseValidDate.AddDate(0, 0, 7)
```

Comment:
why this is changed in your PR?

Reply:
For commercial license , we are adding grace period 30 days after license is expired so that the customer can use the functionalities of the product for extra 30 days but in this period they need to renew the license to continue using it.

Reply:
This is not related to your PR.

Reply:
we have moved this part of code to helper.go in https://github.com/chef/automate/pull/8464 PR, so should I remove this part from infrasture.go of this PR?